docs/synthetics/http-monitor.md

```diff
@@ -0,0 +1,68 @@
+title: HTTP Monitor
+description: HTTP monitor can monitor any HTTP endpoint like API, Web URL, etc. 
+
+The HTTP monitor sends a single HTTP request to the specified URL with configured request settings. It records the response data like headers, body, error if any and the various timings.
+
+### Configuration
+
+#### General
+
+* **Name** - Name of the monitor. Max Length is 255 characters
+* **HTTP Method** - HTTP Method
+* **URL** - URL of the HTTP monitor of format `<http(s)>://hostname:port/path`
+* **Frequency** - Frequency to execute the monitor
+* **Locations** - List of locations to run the monitor
+
+#### Request Settings
+
+* **Headers** - List of HTTP headers to be sent
+* **Query Params** - List of request query parameters to be appended to the URL
+* **Cookies** - List of Cookies to be sent
+* **Body** - Request body data. Max length is 4096 characters
+
+Optionally, the header, query param, and cookie values can be marked a hidden, which means it will be hidden for the users. Currently, we do not encrypt the hidden values while storing it in the DB.
```

Comment:
means it => means they

Btw. by "hidden for the users" do you mean that they will not be displayed in the SC UI?

storing it => storing them